### PR TITLE
FCE-583 Back at it again with the `peer` word

### PR DESCRIPTION
**Pull request by czerwiukk**
## Description

Bring back p word from another p word.

## Motivation and Context

It has turned out to be a simpler approach.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)

### CODE REVIEW

File: packages/react-client/src/types.ts [hunk 1]
```diff
@@ -267,8 +267,8 @@ type DistinguishedTracks = {
 export type PeerStateWithTracks = PeerState & DistinguishedTracks;
 
 export type Participiants = {
```
Comment (kamil-stasiak) [inline packages/react-client/src/types.ts:269]: I found another one with typo

Comment (mironiasty) [reply packages/react-client/src/types.ts:269]: Is this type even required?

**Review by mironiasty — APPROVED** [review]
There is one comment from Kamil
Rest looks good 👍

File: packages/react-client/src/types.ts (hunk 1 above)
Comment (czerwiukk) [reply packages/react-client/src/types.ts]: not used - i removed it

**Pull request merged**